src/odemis/gui/cont/streams.py

```diff
@@ -2621,6 +2660,15 @@ def addCLIntensity(self):
         """ Create a CLi stream and add to to all compatible viewports """
 
         main_data = self._main_data_model
+
+        # Need to pick the right filter wheel (if there is one)
+        axes = {}
+        for fw in (main_data.cl_filter, main_data.light_filter, main_data.tc_od_filter, main_data.tc_filter):
+            if fw is None:
+                continue
+            if main_data.cld.name in fw.affects.value:
+                axes["band"] = ("band", fw)
```

Comment:
`axes["band"]` -> `axes["filter"]`
(for all the streams)
Then you can update gui.conf.data, and drop all `label` line in the config like:
```
            ("band", {  # from filter
                "label": "Filter",
            }),
```